tests/smoke.py

```diff
@@ -0,0 +1,83 @@
+from dataclasses import dataclass
+
+import jax.numpy as jnp  # noqa: F401
+import jupytext
+import numpy as np
+
+
+@dataclass
+class Result:
+    path: str
+    true_mean: float
+    true_stddev: float
+    true_objective: float
```

Comment:
- This is neat. I like it.

- Maybe though there is a better structure that allows for other things we can test. 
- To me the result is just a fixed dataclass of values: 
```python
@dataclass
 class Result:
     true_mean: float
     true_stddev: float
     true_objective: float
```

And we could have smoke test:
```python
@dataclass
 class SmokeTest:
     path: str
     
    def test(self,..., result: Result):
        ....
```
or 
```python
@dataclass
 class SmokeTest:
     path: str
     result: Result

    def test(self,..):
        ....
```
This would perhaps give a bit more flexibility.

Reply:
Sorry, you've lost me here. Why would this be better or more flexible than what we have?

Reply:
how about rewriting it in a way that you can just write
```python
sparse = Result(
    path="docs/examples/collapsed_vi.py",
    history=1924.7634809,
    predictive_mean=-8.39869652,
    predictive_std=255.74838027,
)
sparse.test()

stochastic = Result(
    path="docs/examples/uncollapsed_vi.py",
    history=-985.71726453,
    meanf=-54.14787028,
    sigma=116.16651265,
)
stochastic.test()
```
?

Reply:
(and then it should just be an array instead of its sum for mean & std)